be/src/olap/rowset/alpha_rowset.cpp

```diff
@@ -166,7 +166,9 @@ OLAPStatus AlphaRowset::split_range(
     RowBlockPosition step_pos;
 
     std::shared_ptr<SegmentGroup> largest_segment_group = _segment_group_with_largest_size();
-    if (largest_segment_group == nullptr) {
+    if (largest_segment_group == nullptr || largest_segment_group->current_num_rows_per_row_block() == 0) {
```

Comment:
Why add a check for `largest_segment_group->current_num_rows_per_row_block() == 0`?
You encounter this issue https://github.com/apache/incubator-doris/issues/2750?

Reply:
Yes, so I and a check here to avoid coredump in our product env, but leave a warning log to observe. This problem still needs to be analyzed.

Reply:
OK